frontend/src/app/shared/components/challenges-dropdown-select/challenges-dropdown-select.component.html

```diff
@@ -0,0 +1,24 @@
+<div ngbDropdown class="dropdown d-inline-flex" [autoClose]="true">
```

Comment:
probably this component should be a part of challenge module, is not it?

Reply:
Yea, but the module hasn't been created yet..
https://trello.com/c/lM5A2JG5/35-create-challenge-page I think this task will combine all this components in 1 module

Reply:
But you can create this module =)